export_themes.xml

```diff
@@ -34,6 +34,24 @@
       <section>movies</section>
     </supports>
   </theme>
+  <theme>
+    <name>Hidaya Theme</name>
+    <identifier>hidaya</identifier>
+    <description>beautiful theme</description>
```

Comment:
Could you add a more descriptive text here? e.g. `dark theme with search functionality`?